.browserslistrc

```diff
@@ -1 +1 @@
-> 1% or IE 11
+>= 1%
```

Comment:
I've been contemplating whether we should adjust this to be a bit more accommodating. This number matches what we [prescribe in our documentation](https://github.com/18F/identity-idp/blob/main/docs/frontend.md#devices), but the default behavior of Browserslist will be based on global usage, not usage from our own audience. There are tools like [browserslist-ga](https://github.com/browserslist/browserslist-ga) to use our own analytics, but this would require some setup, since it's doubtful we could (or would want to) connect it directly to Google's APIs ([browserslist-ga-export](https://github.com/browserslist/browserslist-ga-export) is another option).

Maybe we bump it down slightly as a buffer to account for any differences between global usage and our specific usage?

To be clear, we've been using `> 1%` for a while now, but dropping IE11 will be a bit more of a significant jump as far as what syntax/features will be downgraded.

Reply:
I bet we could make a monthly/quarterly task to run either a GA export or a CW export and do that? but I'd say for the time being >1% seems fine

Reply:
`in US` seems better than global, or until we can get our own custom stats +1

Reply:
I went through a few different iterations on this:

- As mentioned above, I first tried `>1% in US` to make it closer to our audience
- After realizing this would exclude territories' users, I tried combining them `>1% in US, >1% in PR, ...`
- After realizing that ">1% in PR" does not reflect ">1% of our users", I contemplated using our own analytics, but was concerned about how to implement this accurately and in a repeatable way that we could maintain long term
- Ultimately, for pragmatic's sake, I arrived back at a percentage-based option

I'm suggesting "0.5%" as a buffer to reflect a concession that despite our desire to support >1% of our own users, Browserslist default data sets would be based on a global audience.